includes/class-api.php

```diff
@@ -1117,6 +1117,11 @@ private function attach_terms( $post_id, $post_args ) {
 		// Set taxonomies for custom post type.
 		if ( isset( $post_args['tax_input'] ) ) {
 			foreach ( $post_args['tax_input'] as $taxonomy => $terms ) {
+				if ( 2 == count( $terms ) ) {
```

Comment:
What is the significance of having 2 terms? This seems pretty obscure without any documentation, either in the form of a meaningful method name (preferred) or a comment about why this check is necessary.